src/livekit/useECConnectionState.ts

```diff
@@ -131,13 +132,20 @@ async function connectAndPublish(
   micTrack: LocalTrack | undefined,
   screenshareTracks: MediaStreamTrack[],
 ): Promise<void> {
+  const tracker = PosthogAnalytics.instance.eventCallConnectDuration;
+  // Track call connect duration
+  tracker.cacheConnectStart();
+  livekitRoom.once(RoomEvent.SignalConnected, () => tracker.cacheWsConnect());
```

Comment:
What if the websocket never connects? Should we clean up this event handler somehow?